packages/color/src/Color.ts

```diff
@@ -207,6 +207,36 @@ export class Color
         return this._int;
     }
 
+    /**
+     * Convert to a hexadecimal number in little endian format (e.g., BBGGRR).
+     * @example
+     * import { Color } from 'pixi.js';
+     * new Color(0xffcc99).toLittleEndianNumber(); // returns 0x99ccff
+     * @returns {number} - The color as a number in little endian format.
+     */
+    toLittleEndianNumber(): number
+    {
+        const value = this._int;
+
+        return (value >> 16) + (value & 0xff00) + ((value & 0xff) << 16);
+    }
+
+    /**
+     * Multiply with another color
+     * @param {PIXI.ColorSource} value - The color to multiply by.
+     */
+    multiply(value: ColorSource): this
+    {
+        const [r, g, b, a = 1] = new Color(value)._components;
```

Comment:
could we use the shared on here also?

```suggestion
        const [r, g, b, a = 1] =  Color.shared.setValue(value)._components;
```

Reply:
I think I tried this and ran into issues if you're already in the shared object. But will try again.